diff --git a/lib/index.js b/lib/index.js
index 66550c9d98e50c8ee96ce95e2f7230d6ad6ce8c8..497f13bd90b6e8ab77f3d97549efa009c554b2e8 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -24,7 +24,7 @@
  * @property {Runtime | null | undefined} [runtime='classic']
  *   Choose the runtime (default: `'classic'`).
  *
- *   Comment form: `@jsxRuntime theRuntime`.
+ *   Comment form: `jsxRuntime: theRuntime`.
  * @property {string | null | undefined} [importSource='react']
  *   Place to import `jsx`, `jsxs`, `jsxDEV`, and `Fragment` from, when the
  *   effective runtime is automatic (default: `'react'`).
